### PR TITLE
Creating virtual machines with vagrant post

### PR DESCRIPTION
**Pull request by sohjiro**
¿Lo podrías revisar manu ( @neodevelop )?

### CODE REVIEW

File: content/post/2017-06-01-creating-virtual-machines-with-vagrant.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
++++
+date = "2017-06-01T15:28:00-05:00"
+author: Felipe Juarez
```
Comment (neodevelop) [inline content/post/2017-06-01-creating-virtual-machines-with-vagrant.md]: Fix:
`author = "Felipe Juarez"`

File: content/post/2017-06-01-creating-virtual-machines-with-vagrant.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+It's been awhile since I wrote a post, but in compensation I would make a series of post talking about deploying applications with edeliver. But for this, we need to prepare the path. So, in this first post I will talk about Vagrant and how to use to setup an environment.
+
+First of all, we're going to install _Vagrant_ and _Virtual Box_. We'll download _Vagrant_ from this [link](https://www.vagrantup.com/downloads.html) and follow the instructions(i.e. next, next, next, agree, install) When that is complete, we can verify the correct installation using the command `vagrant -v`
+
+{{< highlight shell >}}
+❯ vagrant -v
+Vagrant 1.9.3
+{{< /highlight >}}
+
+We can download _Virtual Box_ from this [link](https://www.virtualbox.org). This step is required because _Vagrant_ use images for running in a virtual machine. You can configure _Vagrant_ to be use with other providers, but that is out of this scope.
```
Comment (neodevelop) [inline content/post/2017-06-01-creating-virtual-machines-with-vagrant.md]: “You can configure Vagrant to be **used** with other providers”

**Comment by neodevelop** [comment]
That's all!
Great, brief and useful!

Regards.

**Comment by sohjiro** [comment]
Thx @neodevelop I already fix it

**Pull request merged**